templates/functional.html.j2

```diff
@@ -108,7 +108,7 @@
 <p>The function is available in the following states of the {{ owner | safe }}:</p>
 <ul>
     {% for state in object.available_in_states %}
-        <li>{{linked_name_with_icon(state) | safe}}</a></li>
+        <li><a>{{linked_name_with_icon(state) | safe}}</a></li>
```

Comment:
Here too, the closing tag was left over from previous code and should be removed instead. An `<a>` without any attributes is pretty much useless anyway, and `linked_name_with_icon` produces another `<a>` which would be nested within that.